WorkloadManagementSystem/DB/JobDB.sql

```diff
@@ -107,6 +107,17 @@ CREATE TABLE `JobParameters` (
   FOREIGN KEY (`JobID`) REFERENCES `Jobs`(`JobID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+-- ------------------------------------------------------------------------------
+DROP TABLE IF EXISTS `JobsStatus`;
+CREATE TABLE `JobsStatus` (
+  `JobID` INT(11) UNSIGNED NOT NULL,
+  `Status` VARCHAR(32) NOT NULL DEFAULT 'Received',
+  `MinorStatus` VARCHAR(128) NOT NULL DEFAULT 'Unknown',
+  `ApplicationStatus` VARCHAR(255) DEFAULT 'Unknown',
+  PRIMARY KEY (`JobID`),
+  FOREIGN KEY (`JobID`) REFERENCES `JobJDLs`(`JobID`)
```

Comment:
It's the first time I see a primary key be a foreign key at the same time. I understand the reasons of doing it, but out of luck, did you check the literature on that ? Whether it is recommended or not ?

Reply:
Yes, I looked up for it. Even I was quite surprised to see its usage in other tables. Though it's not recommended (UNIQUE usage is suggested).